modular_skyrat/modules/importedvendors/code/vendor_snacks.dm

```diff
@@ -0,0 +1,275 @@
+
+/obj/item/reagent_containers/cup/glass/dry_ramen/prepared/hell
+	name = "spicy cup ramen"
+	desc = "This one comes with water, AND a security checkpoint's worth of capsaicin!"
+	list_reagents = list(/datum/reagent/consumable/hell_ramen = 15, /datum/reagent/consumable/salt = 3)
+
+/obj/item/food/vendor_snacks
+	name = "\improper God's Strongest Snacks"
+	desc = "You best hope you aren't a sinner. (You should never see this item please report it)"
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "foodpack_generic"
+	trash_type = /obj/item/trash/vendor_trash
+	bite_consumption = 10
+	food_reagents = list(/datum/reagent/consumable/nutriment = INFINITY)
+	junkiness = 10
+	custom_price = PAYCHECK_LOWER * INFINITY
+	tastes = list("the unmatched power of the sun" = 10)
+	foodtypes = JUNKFOOD | CLOTH | GORE | NUTS | FRIED | FRUIT //You don't want to know what's in the broken debug snacks
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/trash/vendor_trash
+	name = "\improper God's Weakest Snacks"
+	desc = "The leftovers of what was likely a great snack in a past time."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_quick_foods.dmi'
+	icon_state = "foodpack_generic_empty"
+	custom_materials = list(/datum/material/plastic=1000)
+
+/obj/item/food/vendor_snacks/ricecrackers
+	name = "rice crackers"
+	desc = "Despite most of the package being clear, you will never truly know what flavor these are until you eat them."
+	icon_state = "ricecracka"
+	trash_type = /obj/item/trash/vendor_trash/ricecracka
+	food_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/consumable/rice = 2)
+	tastes = list("incomprehensible flavoring" = 1, "rice cracker" = 2)
+	foodtypes = JUNKFOOD | GRAIN
+	custom_price = PAYCHECK_LOWER * 0.8
+
+/obj/item/food/vendor_snacks/ricecrackers/MakeLeaveTrash()
+	AddElement(/datum/element/food_trash, trash_type, FOOD_TRASH_POPABLE)
+
+/obj/item/trash/vendor_trash/ricecracka
+	name = "empty rice crackers bag"
+	desc = "You never did find out what flavor that was supposed to be, did you?"
+	icon_state = "ricecracka_trash"
+
+/obj/item/food/vendor_snacks/mochi_icecream
+	name = "mochi icecream balls - vanilla"
+	desc = "A six pack of mochi icecream, which is to say vanilla icecream surrounded by mochi. Comes with small plastic skewer for consumption."
+	icon_state = "mochiice"
+	trash_type = /obj/item/trash/vendor_trash/mochi_icecream
+	food_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/ice = 3)
+	tastes = list("rice cake" = 2, "vanilla" = 2)
+	foodtypes = JUNKFOOD | DAIRY | GRAIN
+	custom_price = PAYCHECK_LOWER * 1.2
+
+/obj/item/food/vendor_snacks/mochi_icecream/matcha
+	name = "mochi icecream balls - matcha"
+	desc = "A six pack of mochi icecream, which is to say matcha icecream surrounded by mochi. Comes with small plastic skewer for consumption."
```

Comment:
```suggestion
	desc = "A six pack of mochi icecream - or, more specifically, matcha icecream surrounded by mochi. Comes with small plastic skewer for consumption."
```
I dont know if this is grammatically correct, but it makes more sense to me; I had to google each component to make sense of the old description, but I think I was just thrown off by the term 'which is to say'

Also during my research, I legit couldn't conclude if it's meant to be icecream, ice cream, or ice-cream. So we'll juuuuust leave that